browse/services/database/models.py

```diff
@@ -293,6 +295,93 @@ class DBLPDocumentAuthor(db.Model):
     document = relationship('Document')
 
 
+class Category(db.Model):
```

Comment:
What is the plan for these categories in the DB and the taxonomy in arxiv-base?

Reply:
None yet. For the time being they will have to coexist as before.